src/service/video-manager-impl.js

```diff
@@ -635,6 +651,7 @@ class VideoEntry {
 
     if (this.getPlayingState() == PlayingStates.PLAYING_MANUAL) {
       this.firstPlayEventOrNoop_();
+      this.manager_.pauseOtherVideos(this);
```

Comment:
You might need to do this on an `UNMUTE` signal for autoplaying videos.

Reply:
Added, good point!